Source/WebDriver/Session.cpp

```diff
@@ -3176,29 +3176,44 @@ void Session::takeScreenshot(std::optional<String> elementID, std::optional<bool
 void Session::dispatchBidiMessage(RefPtr<JSON::Object>&& message)
 {
     // Validate bidi message
+    if (!message) {
+        RELEASE_LOG(WebDriverBiDi, "Session::dispatchBidiMessage: Invalid message parameter. Ignoring.");
+        return;
+    }
+
     auto params = message->getObject("params"_s);
+    RefPtr<WebSocketServer> protectedBidiServer = m_bidiServer.get();
```

Comment:
I don’t think we need such a long name for this local variable. Sure, it’s properly referenced, but I don’t think we benefit from expressing that in the name. And in the context of a function dispatchBidiMessage we don’t need to emphasize this is the “bidi” server in the name either.
```suggestion
    RefPtr server = m_bidiServer.get();
```